dev/src/index.ts

```diff
@@ -1721,3 +1350,14 @@ Object.defineProperty(module.exports, 'v1', {
     return v1;
   },
 });
+
+Object.defineProperty(module.exports, 'GrpcStatus', {
```

Comment:
We use the same technique for two other exports. Right now, we still produce `commonjs` though. I suspect this might not work with ESM, but we can cross that bridge when we get there (I don't have a suggestion though)